cli/main.go

```diff
@@ -300,20 +300,21 @@ $ docker context create %s <name>`, cc.Type(), store.EcsContextType), ctype)
 
 func exit(ctx string, err error, ctype string) {
 	if exit, ok := err.(cli.StatusError); ok {
-		metricsClient.Track(ctype, os.Args[1:], compose.SuccessStatus)
+		// TODO(milas): shouldn't this use the exit code to determine status?
+		metricsClient.Track(ctype, os.Args[1:], metrics.SuccessStatus)
```

Comment:
I agree, we should use the exit code and send the associated status IHMO

Reply:
Yeah, I believe this is a long-standing bug 😅 

I'll confirm today and open a quick PR that just adds the exit code -> status wrapper call here, we have all the info available, so I think this was a copy-paste mistake rather than intentional